theme/breadcrumbs.html

```diff
@@ -26,12 +26,15 @@
           {% endif %}
         {% endset %}
 
-        {% if "actions/" in page.file.src_path %}
-          <a href="{{ config.fastlane_repo_base_edit_uri }}/{{ page.file.src_path|string|safe|replace(".md", ".rb") }}" {{ icon_class_block }}>
-        {% elif page.edit_url %}
-          <a href="{{ page.edit_url }}" {{ icon_class_block }}>
-        {% endif %}
-        Edit on {{ config.repo_name }}</a>
+        {% set edit_docs_file_url %}
+          {%- if "actions/" in page.file.src_path %}
+            {{ config.fastlane_repo_base_edit_uri }}/{{ page.file.src_path|string|safe|replace(".md", ".rb") }}
+          {%- elif page.edit_url %}
+            {{ page.edit_url }}
+          {%- endif %}
+        {% endset %}
+
+        <a href="{{ edit_docs_file_url }}" {{ icon_class_block }}>Edit on {{ config.repo_name }}</a>
```

Comment:
This would be better:

```suggestion
        <a href="{{ edit_docs_file_url }}" {{ icon_class_block }}> Edit on {{ config.repo_name }}</a>
```

<img  alt="Screen Shot 2021-11-21 at 2 21 33" src="https://user-images.githubusercontent.com/1784648/142745734-6dc2ab72-c095-4b55-a131-37e5a33c14b2.png">

I know, it's weird from a semantic point of view to have the space an integral part of the anchor text… however that's how the original sphinx/rtd template does it to prevent exactly this aligment issue. 🤷